onnx/backend/test/runner/__init__.py

```diff
@@ -141,14 +146,14 @@ def tests(self):
         return tests
 
     @staticmethod
-    def _assert_similar_outputs(ref_outputs, outputs):
+    def _assert_similar_outputs(ref_outputs, outputs, rtol, atol):
```

Comment:
Maybe do `rtol=None` and `atol=None` and set them to the default value if not specified? So people don't have to override both of them.

Reply:
I tag the wrong line, I meant in `def tolerance`, not `def _assert_similar_outputs`. But looks like you have done the right change :-)